docs/guides/admin/docs/configuration/inbox.md

```diff
@@ -0,0 +1,121 @@
+InboxScannerService
+=====================
+
+Overview
+--------
+
+Besides ingesting media packages using the REST service of the IngestService,
+dedicated inbox directories located in the file system can be scanned by
+Opencast. This, for example allows adding media packages to Opencast by copying
+it to a specific location using scripting/SFTP without the need for any HTTP
+traffic. Opencast periodically scans the specified location for new files.
+
+Each directory may result in digest for a different organization or with a
+different default workflow.
+
+Step 1: Configure an InboxScannerService
+-----------------------------------------------
+
+Adjust
+[etc/org.opencastproject.ingest.scanner.InboxScannerService-inbox.cfg](`https://github.com/opencast/opencast/blob/develop/etc/org.opencastproject.ingest.scanner.InboxScannerService-inbox.cfg`).
+
+The `-inbox` suffix of the file name is variable and multiple files can
+be created with different settings for different directories to be watched.
+
+
+Step 2: Testing the inbox
+-----------------------------------------
+
+In order to test the inbox scanner service, either put valid media package
+zip into the scanned directory or a single media file.
+
+Note that even if the poll interval is small, it may take a little longer until
+the media package is added to the epsisode admin UI view because extract and
+copy operations may take some time.
+
+Example media package
+-----------------------------------------
+
+Media packages contain media files and metadata files describing them.
+Opencast is able to generate media packages using the
+[ZipWorkflowOperation](../workflowoperationhandlers/zip-woh/).
+
+Create the follwing files:
+
+_manifest.xml_
+```
+<?xml version="1.0" encoding="utf-8"?>
+<mediapackage xmlns:oc="http://mediapackage.opencastporject.org">
+<title>A media package courtesy by the inbox scanner.</title>
```

Comment:
This title should probably match the dublin core title.

Reply:
Changing dublin core title.